From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ManaviS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@ManaviS Thank you for this contribution.  I'll have the author review.

Commit:
Updated for GW in hub

danieldobalian committed:
Updated titles to mirror metadata tags for quickstarts

Ja-Dunn committed:
edit pass: monitoring-service-notifications

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into monitoring-service-notifications

Ja-Dunn committed:
edit pass: monitoring-service-notifications

YuriDio committed:
updating article

danieldobalian committed:
cleaning up pr

danieldobalian committed:
Merge branch 'master' into title-update

danieldobalian committed:
fixing erequested edits

danieldobalian committed:
resoling conflicts

danieldobalian committed:
changing spacing

Commit:
new file with screenshots and updated TOC

Commit:
formatting and acrolinx fixes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic1222

klaaslanghout committed:
Update storage-sync-files-planning.md

Add info on no-rest changes

klaaslanghout committed:
Update storage-sync-files-change-detection.md

Adding the fact that REST updates are not synced

mmacy committed:
[aci] refresh + link maintenance

mmacy committed:
[aci] output update

mmacy committed:
[aci] term correction

mmacy committed:
[aci] param ordering

mmacy committed:
[aci] minor fix

mmacy committed:
[aci] heading concision

kpiteira committed:
Updated files to remove old portal screenshot

Commit:
fix javareliableservicesoverview

Commit:
fix javareliableservicesoverview

Commit:
Merge branch 'javareliableservicesoverview' of https://github.com/suhuruli/azure-docs-pr into javareliableservicesoverview

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into UpdatesDez12

Commit:
Updated AADconnect TASK 1147988

Commit:
update to text

Commit:
documenting the -c switch in az ml service create realtime

Commit:
documenting the -c switch in az ml service create realtime

Commit:
update after review

Commit:
whatsnew04

kpiteira committed:
Changed some line spacing

kpiteira committed:
Changed some line spacing

danieldobalian committed:
fixing merge conflicts

Commit:
whatsnew04

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into UpdatesDez12

YuriDio committed:
updating article as per PM request

Commit:
whatsnew04

YuriDio committed:
fixing acro

ShawnJackson committed:
Merge pull request #29217 from Ja-Dunn/monitoring-service-notifications

edit pass: monitoring-service-notifications

ktoliver committed:
fix code issue

cwatson-cat committed:
Update author on HDInsight topics to retire topic.

YuriDio committed:
fixing acro 2

ktoliver committed:
remove extra line before code

Lisaco88 committed:
Merge pull request #30310 from mmacy/aci-link-formatting

[aci] refresh + link maintenance

ktoliver committed:
Merge pull request #28971 from danieldobalian/title-update

Updated titles to mirror metadata tags for quickstarts

ktoliver committed:
Merge pull request #30332 from kpiteira/azme-remove-oldportal

Updated files to remove old portal screenshot

v-geberr committed:
new ux for add entities

v-geberr committed:
link to fuzzy search

v-geberr committed:
add entity

ktoliver committed:
Merge pull request #29960 from YuriDio/UpdatesDez12

updating article

Lisaco88 committed:
Merge pull request #30364 from hning86/master

document the -c switch in az ml service create realtime in iris part 3

v-geberr committed:
touch

v-thepet committed:
fix broken links

wmgries committed:
Revert "Updating vers=3.0 to vers=2.1"

This reverts commit fe51c2f5a8b9fbf6268d221050e848dde1a0e4fa.

Commit:
whatsnew04

fimguy committed:
Update active-directory-aadconnectsync-connector-genericsql.md

ktoliver committed:
Merge pull request #30374 from cwatson-cat/12-19-RetireHDInsightTopic

Update author on HDInsight topics to retire topic.

anrothMSFT committed:
api reference now a forwardlink

carolinacmoravia committed:
C8464: Hardcoded EN locale in link

Hello, @YuriDio,

Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?

Many thanks in advance.

ktoliver committed:
Merge pull request #30359 from fimguy/davidste-connectordupdates

Updated AADconnect TASK 1147988

carolinacmoravia committed:
C8459: Hardcoded EN locale in link 

Hello, @mawah,

Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?

Many thanks in advance.

Commit:
update after review

ktoliver committed:
Merge pull request #30368 from MarkusVi/whatsnew04

whatsnew04

v-geberr committed:
add features new ux

cwatson-cat committed:
Retired HDInsight topic, deleted images, updated references to topic.

v-geberr committed:
add features - fix image

Lisaco88 committed:
Merge pull request #30292 from craigcaseyMSFT/v-craic1222

new file with screenshots and updated TOC

Commit:
use hashmap instead of collection and fix code snippet

Ja-Dunn committed:
Merge pull request #30336 from suhuruli/javareliableservicesoverview

Javareliableservicesoverview

Ja-Dunn committed:
Merge pull request #30380 from v-thepet/linkfix

production bug 71650 fix broken links

Ja-Dunn committed:
Merge pull request #30379 from v-geberr/1218-entities-screenshots

1218 entities screenshots

GitHubber17 committed:
Merge pull request #30384 from carolinacmoravia/patch-521

C8464: Hardcoded EN locale in link

Ja-Dunn committed:
Merge pull request #30382 from anrothMSFT/patch-1

api reference now a fwlink

GitHubber17 committed:
Merge pull request #30385 from carolinacmoravia/patch-522

C8459: Hardcoded EN locale in link

anrothMSFT committed:
added compact domains to glossery

Juliako committed:
Update media-services-release-notes.md

Juliako committed:
Update media-services-concepts.md

Ja-Dunn committed:
Merge pull request #30386 from v-geberr/1219-add-features

1219 add features

anrothMSFT committed:
added compact domains

Ja-Dunn committed:
Merge pull request #30389 from anrothMSFT/patch-2

added compact domains to glossary

GitHubber17 committed:
Merge pull request #30387 from cwatson-cat/12-19-RetireHDInsightTopic

Retired HDInsight topic, deleted images, updated references to topic.

Ja-Dunn committed:
Merge pull request #30390 from Juliako/patch-2

Update media-services-release-notes.md

Ja-Dunn committed:
Merge pull request #30391 from Juliako/patch-3

Update media-services-concepts.md

Ja-Dunn committed:
Merge pull request #30394 from anrothMSFT/patch-3

added compact domains

anrothMSFT committed:
release notes for 12/19 release

klaaslanghout committed:
added <br /> < br /> to add breaking line space

wmgries committed:
Clarify wording around SMB/CIFS support.

klaaslanghout committed:
Update storage-sync-files-planning.md

Ja-Dunn committed:
Merge pull request #30308 from klaaslanghout/patch-6

Update storage-sync-files-change-detection.md

Commit:
Merge pull request #30307 from klaaslanghout/patch-4

Update storage-sync-files-planning.md

anrothMSFT committed:
Update home.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #26651 from jimdial/patch-18

Updated for GW in hub

Ja-Dunn committed:
Merge pull request #30401 from wmgries/linux-fixup

Technical corrections for Azure Files with Linux doc

v-alje committed:
Resolving merge conflict

Ja-Dunn committed:
Merge pull request #30400 from anrothMSFT/patch-5

release notes for 12/19 release

v-alje committed:
Merge pull request #30403 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
